client/packages/common/src/utils/formatters/JSXFormatters.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export const JSXFormatters = {
+  /*
+  Converts a string with line breaks (\n) into HTML with <br/> line breaks
+  */
+  replaceHTMLlineBreaks: (input: string) => {
+    const lines = input.split('\n');
+    return lines.map((line, i) => (
+      <span key={i}>
+        {line}
+        {i === lines.length - 1 ? null : <br />}
+      </span>
+    ));
+  },
+};
```

Comment:
This seemed like a useful utility that might come in handy elsewhere, so I've put it in a new utils file. Not sure of the best place for this, but didn't quite seem to fit in with the other utils. Any particular suggestions, @mark-prins ?